docs/changelog.rst

```diff
@@ -22,12 +22,13 @@ v0.3.4.dev0
 
 * :gh:issue:`929` Support Ansible 6 and ansible-core 2.13
 * :gh:issue:`832` Fix runtime error when using the ansible.builtin.dnf module multiple times
-* :gh:issue:`925` :class:`ansible_mitogen.connection.Connection` no longer tries to close the 
```

Comment:
Please don't modify lines unrelated to your change. Even just whitespace

Reply:
Fixed this, sorry for the noise.

Reply:
Fixed this, sorry for the noise.